src/Orchard.Web/Web.config

```diff
@@ -15,6 +15,12 @@
         <add key="webpages:Version" value="3.0.3"/>
         <add key="log4net.Config" value="Config\log4net.config"/>
         <add key="owin:AppStartup" value="Orchard.Owin.Startup, Orchard.Framework"/>
+    
+<!-- optional additional comma-separated list of folders for Modules or Themes -->
+        <add key="Modules" value="~/Ibn_Modules"/>
+<!--
+        <add key="Themes" value="~/Themes/MoreThemes" />
+-->

```

Comment:
Sebastien also mentioned under the issue that it's probably best to have this config in the HostComponents.config file and consume it similarly how the services do it there.